src/ontology/nbo-edit.owl

```diff
@@ -10796,6 +10801,7 @@ owl:qualifier some &apos;temporally extended&apos;)</oboInOwl:note>
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/NBO_0000645">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/NBO_0000643"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/NBO_0010000"/>
```

Comment:
Didnt you want to remove the subClassOf behaviour phenotype assertions as well?

Reply:
I think it would be less disruptive of any existing users to keep it.  Where there is a logical def, phenotypes use an EQ pattern, rather than QE as everyone else does, so no formal upper ontology issue I think.

Reply:
Yeah but we will pull in that "behaviour phenotype" class - I was asked to not only remove subclasses of that, but the entire branch, to avoid confusion..

Reply:
not just bottom - any module extract that will pull in parents!